Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -364,6 +355,9 @@ - (UIBarButtonItem*)shareToolbarItem {
 - (void)viewDidLoad {
     [super viewDidLoad];
 
+    [self setupToolbar];
```

Comment:
ah sorry, misread the diff. still, it needed to be in `init` because that guarantees `toolbarItems` isn't empty, which allows the toolbar to be shown. unless you want to change how toolbar appearance/disappearance works, it needs to stay in `init, which is [what Apple recommends](https://developer.apple.com/library/prerelease/tvos/documentation/WindowsViews/Conceptual/ViewControllerCatalog/Chapters/NavigationControllers.html#//apple_ref/doc/uid/TP40011313-CH2-SW26).